stage-module/00-i2c-tweaks/00-run.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash -e
+
+CONFIG=${ROOTFS_DIR}/boot/config.txt
+
+printf "\n# Enable I2C0 on pins 44, 45" | tee -a $CONFIG
```

Comment:
I think you should not `tee` them to stdout, looks ungly in the build log:
![image](https://user-images.githubusercontent.com/8187156/93989641-06f97480-fd3f-11ea-9fb8-9585266bb978.png)